src/tools/xkbctrl

```diff
@@ -116,6 +88,9 @@ sub ReadDataConfigMap {
 		die "could not open file: $filename";
 	}
 	while (my $line=<DATA>) {
+		$line =~ s/\t+/ /g;
+		$line =~ s/ +/ /g;
+		$line =~ s/ /:/g;
```

Comment:
On Tue, Aug 18, 2015 at 07:17:46AM -0700, Ancor Gonzalez Sosa wrote:
> In src/tools/xkbctrl:
> 
> > @@ -116,6 +88,9 @@ sub ReadDataConfigMap {
> >               die "could not open file: $filename";
> >       }
> >       while (my $line=<DATA>) {
> > +             $line =~ s/\t+/ /g;
> > +             $line =~ s/ +/ /g;
> > +             $line =~ s/ /:/g;
> 
> Ok, if @sndirsch agrees, I'll reduce it to one line.

Sure. As long as you don't break the semantic, go ahead!

Thanks,
Stefan

Public Key available
------------------------------------------------------
Stefan Dirsch (Res. & Dev.)   SUSE LINUX GmbH
Tel: 0911-740 53 0            Maxfeldstraße 5
FAX: 0911-740 53 479          D-90409 Nürnberg
http://www.suse.de            Germany 
---------------------------------------------------------------
SUSE Linux GmbH, GF: Felix Imendörffer, Jane Smithard, Graham
Norton, HRB 21284 (AG Nürnberg)
---------------------------------------------------------------